docs/support.md

```diff
@@ -0,0 +1,13 @@
+# Support
+
+By contacting us using our [online portal](https://anchorfleet.zendesk.com/) we can provide you with an auditable response that will provide *transparency* to all parties involved. For *incident form requests* we ask the following so we can resolve your query as quickly as possible:
+
+* Attempted action
+* Expected Result
```

Comment:
Should be consistent here.

Either use sentence case for all lines (Attempted action, Expected result) or capitalise all words (Attempted Action, Expected Result)